keyboards/epomaker/tide65/info.json

```
@@ -0,0 +1,237 @@
+{
+    "manufacturer": "HS",
+    "keyboard_name": "EPOMAKER TIDE 65",
+    "maintainer": "sdk66",
+    "bootloader": "wb32-dfu",
+    "bootmagic": {
+        "matrix": [0, 0]
+    },
+    "diode_direction": "ROW2COL",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B7", "pin_b": "B6"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
```

Probably want this here.
```suggestion
        "nkro": true,
        "rgb_matrix": true
```